packages/playwright-core/src/server/electron/electron.ts

```diff
@@ -73,28 +77,60 @@ export class ElectronApplication extends SdkObject {
       if (!event.context.auxData || !event.context.auxData.isDefault)
         return;
       const crExecutionContext = new CRExecutionContext(this._nodeSession, event.context);
-      this._nodeExecutionContext = new js.ExecutionContext(this, crExecutionContext, 'electron');
       const { result: remoteObject } = await crExecutionContext._client.send('Runtime.evaluate', {
         expression: `require('electron')`,
         contextId: event.context.id,
         // Needed after Electron 28 to get access to require: https://github.com/microsoft/playwright/issues/28048
         includeCommandLineAPI: true,
       });
+      this._nodeExecutionContext = new js.ExecutionContext(this, crExecutionContext, 'electron');
```

Comment:
I think if you bring this line back before the `await`, it is guaranteed that execution context exists before any `Runtime.consoleAPICalled` events may arrive.